js/src/types.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-env mocha */
+'use strict'
+
+const PeerId = require('peer-id')
+const PeerInfo = require('peer-info')
+const dagCBOR = require('ipld-dag-cbor')
+const dagPB = require('ipld-dag-pb')
+const multiaddr = require('multiaddr')
+const multibase = require('multibase')
+const multihash = require('multihashes')
+const CID = require('cids')
+
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+const expect = chai.expect
+chai.use(dirtyChai)
+
+describe('.types', function () {
```

Comment:
Are these runnable? There's no `module.exports = (common) => {` wrapper and no entry in `js/src/index.js` for them.

Reply:
@vasco-santos did you open a new PR?

Reply:
It is created now 🙂